dataactvalidator/app.py

```diff
@@ -1,111 +1,73 @@
 import logging
 
-import csv
-from flask import Flask, request, g
+from flask import Flask, g, current_app
 
 from dataactcore.config import CONFIG_BROKER, CONFIG_SERVICES
 from dataactcore.interfaces.db import GlobalDB
-from dataactcore.interfaces.function_bag import write_file_error, mark_job_status
 from dataactcore.logging import configure_logging
-from dataactcore.models.jobModels import Job
-from dataactcore.utils.jsonResponse import JsonResponse
-from dataactcore.utils.responseException import ResponseException
-from dataactvalidator.validation_handlers.validationError import ValidationError
 from dataactvalidator.validation_handlers.validationManager import ValidationManager
+from dataactcore.models.jobModels import SQS
+from dataactcore.aws.sqsHandler import get_queue
 
 
 logger = logging.getLogger(__name__)
+logging.getLogger('boto3').setLevel(logging.CRITICAL)
+logging.getLogger('botocore').setLevel(logging.CRITICAL)
```

Comment:
Probably want to put this in the config file (or add it to `dataactcore.logging`). Just be sure that that `configure_logging` is called somewhere.